system-addon/lib/ManualMigration.jsm

```diff
@@ -18,7 +17,22 @@ XPCOMUtils.defineLazyModuleGetter(this, "ProfileAge", "resource://gre/modules/Pr
 this.ManualMigration = class ManualMigration {
   constructor() {
     Services.obs.addObserver(this, MIGRATION_ENDED_EVENT);
-    this._prefs = new Prefs();
+  }
+
+  get getMigrationLastShownDate() {
+    return this.store.getState().Prefs.values.migrationLastShownDate;
+  }
+
+  set setMigrationLastShownDate(newDate) {
+    this.store.getState().Prefs.values.migrationLastShownDate = newDate;
+  }
+
+  get getMigrationRemainingDays() {
+    return this.store.getState().Prefs.values.migrationRemainingDays;
```

Comment:
Thanks for cleaning this up. I think instead of having individual getter/setters for each pref, it would be simple enough to have a helper that takes in the pref name. Although at that point, it's nothing specific to manual migration, so maybe it should be available for all Feeds?

@k88hudson any thoughts on having `this.store.getPref(name)` and similar `setPref` helper that dispatches?

Reply:
I can see that being helpful, yeah; I guess it is a bit verbose to do `this.store.getState().Prefs.values` every time.

I think extending `store` to have `store.getPref` and `store.setPrefs` as helpers would be ok, but if it's out of the scope of this PR a single getter/setter in `ManualMigration.jsm` would be fine by me too.